obevo-site/src/site/markdown/db-project-structure.md

```diff
@@ -726,7 +726,18 @@ COLUMN_ID, COLUMN_NAME, ADD_TIME
 
 Just define a CSV file (quotes are supported, as is changing the comma
 delimiter and null token) with the first row as the column names, and
-you are set
+you are set.
+
+By default, as shown above, the literal `null` (without surrounding quotes)
+is interpreted as the null value. If you wish to modify that, you can
+do so using the `nullToken` property in the `METADATA` header as follows:
+
+```
+//// METADATA nullToken="myNullToken"
+```
+
+When providing the null token in the CSV content, whether the default or custom, 
+it should not have surrounding quotes.
```

Comment:
want to confirm this is actually correct

Reply:
Yes and no :) - I've removed those lines from this review, but potentially some other doc is needed, which I can try in a separate ticket

Some background:

The original CSV handling used within GS is using csvVersion 1. It had a few issues around quoting, multi-lines, and such.

I created a version 2 that handles this much more elegantly.

However, for backwards-compatibility, I could not update the GS distribution to use V2 by default. Public users however do have v2 enabled by default. You can see the logic flipping in the class CsvReaderDataSource